api/http/handler/settings/settings_update.go

```diff
@@ -125,6 +135,10 @@ func (handler *Handler) settingsUpdate(w http.ResponseWriter, r *http.Request) *
 		settings.EdgeAgentCheckinInterval = *payload.EdgeAgentCheckinInterval
 	}
 
+	if payload.UserSessionTimeout != nil {
+		settings.UserSessionTimeout = *payload.UserSessionTimeout
```

Comment:
So are we not propagating the changes to the JWTService here? How can it be updated then? See my comment in previous review about updating the JWTService interface to allow the update of the user session timeout.

Reply:
my mistake forgot to push it